lib/Doctrine/DBAL/Schema/Schema.php

```diff
@@ -282,23 +245,19 @@ public function getSequence($sequenceName)
     }
 
     /**
-     * @return Sequence[]
+     * @return array<string, Sequence>
      */
-    public function getSequences()
+    public function getSequences() : array
     {
         return $this->_sequences;
     }
 
     /**
      * Creates a new namespace.
      *
-     * @param string $namespaceName The name of the namespace to create.
-     *
-     * @return \Doctrine\DBAL\Schema\Schema This schema instance.
```

Comment:
It'd make sense to keep it annotated as `@return $this`. See #2384 for the reference.

Reply:
Fixed in https://github.com/doctrine/dbal/pull/3565/commits/05eca9944e37463a6ff0636c8c1c89a081321402